### DISCUSSION TITLE
Go migration files are not migrated

### DISCUSSION THREAD

**Discussion by kronnakrit** [pocketbase/pocketbase#5297] [Q&A] [answered]
Hi all,

I tried to migrate Go migration file with command `go run main.go migrate` but I what I got from migrating is image below

<img width="655" alt="image" src="https://github.com/user-attachments/assets/b84ce3ec-e5d2-4752-b6b8-548065ae051e">

And this is what I have in my migrations folder

<img width="351" alt="image" src="https://github.com/user-attachments/assets/88a5a53f-36be-4f31-ae2a-1329c06f94e7">

So, I tried many things but I have no idea why this is happening. Migrating with JS is working fine for me with command `pocketbase migrate` but it's not working with customer build one.

Image below is my main.go 

<img width="701" alt="image" src="https://github.com/user-attachments/assets/2b07b5a0-abac-481e-a8d0-a71249e2c40a">

Any idea?

**Comment by ganigeorgiev** [comment, marked as answer]
_Please don't post code as screenshot. It is difficult to read and copy._

You need to import your `migrations` package so that Go could be aware of the files (_I'll consider maybe adding a warning for this in the future_).

Or in other words you can try to add:

```go
import (
  ...
  _ "yourApp/migrations" // adjust based on your module and the migrations dir location
)
```

**Reply by kronnakrit** [reply]
Thank you!! I will keep that in mind about post code as screenshot.

**Reply by siraphobk** [reply]
Thank you!
